epios/tests/test_post_process.py

```diff
@@ -0,0 +1,559 @@
+import numpy as np
+import pandas as pd
+import unittest
+from unittest import TestCase
+from unittest.mock import patch
+from post_process import PostProcess
+import os
+import sys
+# from numpy.testing import assert_array_equal
+from pandas.testing import assert_frame_equal
+
+
+class TestDataProcess(TestCase):
+
+    def setUp(self) -> None:
+        '''
+        This function set up the testing environment
+        Firstly use the DataProcess class to generate input for Sampler
+        Secondly construct the Sampler class
+
+        This function include some of the expected results
+
+        '''
+        self.path = './testing_post_process_mod/'
+        try:
+            os.mkdir(self.path[2:-1])
+        except FileExistsError:
+            raise FileExistsError('Directory already exists, terminated not to overwrite anything!')
+        self.demo_data = pd.DataFrame({'ID': ['0.0.0.0', '0.0.0.1', '0.0.1.0',
+                                              '0.1.0.0', '0.2.0.0', '1.0.0.0'],
+                                       'age': [1, 81, 45, 33, 20, 60]})
+        self.time_data = pd.DataFrame({'time': [0, 1, 2, 3, 4, 5],
+                                       '0.0.0.0': ['InfectASympt'] * 6,
+                                       '0.0.0.1': [0, 0, 0, 'InfectASympt', 'InfectASympt', 'InfectASympt'],
```

Comment:
Do you still want the names rather than numbers here, as I think epiabm always outputs numbers